.github/workflows/CI.yml

```diff
@@ -28,5 +28,7 @@ jobs:
         java-version: ${{ matrix.java }}
 
     - name: Build and Run Tests
+      # The following publishPluginZipPublicationToZipStagingRepository task is generated by 'opensearch.pluginzip' custom gradle plugin
```

Comment:
Nit: can we move the comment below and just say `# generated by ...`

Reply:
Just pushed this in my latest commit.